.github/workflows/container-image.yml

```
@@ -82,11 +85,36 @@ jobs:
           cargo build --release --target ${{ matrix.targetarch }}-unknown-linux-musl
           mv target/${{ matrix.targetarch }}-unknown-linux-musl/release/policy-server policy-server-${{ matrix.targetarch }}
 
-      - name: Upload policy-server
+      - name: Generate SBOM
+        run: |
+          make sbom
+          # SBOM files should have "sbom" in the name due the CLO monitor
+          # https://clomonitor.io/docs/topics/checks/#software-bill-of-materials-sbom
+          cp bom-cargo.json policy-server-${{ matrix.targetarch }}-sbom.spdx.json
```

I personally prefer to move it:

```suggestion
          mv bom-cargo.json policy-server-${{ matrix.targetarch }}-sbom.spdx.json
```